gear-contract-libraries/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "gear-contract-libraries"
+version = "0.1.0"
+edition = "2021"
+license = "GPL-3.0"
+authors = ["Gear Technologies"]
+
+[lib]
+name = "gear_contract_libraries"
+path = "lib.rs"
+crate-type = [
+    "rlib",
+]
```

Comment:
This seems to be by default. May we remove it?
```suggestion
```

Reply:
After removing you will get: 
Caused by:
  no targets specified in the manifest
  either src/lib.rs, src/main.rs, a [lib] section, or [[bin]] section must be present